tests/test_contextual_orchestrator_review_runtime_preflight.py

```
@@ -1,1770 +1,37 @@
-        assert Path(value).exists()
-        raise RuntimeError("loader rejected catalog")
-
-    with pytest.raises(RuntimeError, match="loader rejected catalog"):
-        helper(str(path), agents, loader=failing_loader)
-    assert not path.exists()
-
-
-def test_preflight_transport_has_no_inference_timeout_and_is_provider_neutral() -> None:
+def test_preflight_transport_has_no_inference_timeout_and_uses_bounded_retry() -> None:
+    """Review inference is deadline-free while preflight retries once."""
     launcher = _LAUNCHER.read_text(encoding="utf-8")
```

Ruff (0.16.3)

[error] Undefined name `_LAUNCHER`

(F821)